src/client/Microsoft.Identity.Client/Platforms/Android/Broker/AndroidBrokerHelper.cs

```diff
@@ -208,31 +120,30 @@ public void CheckForBrokerAccountInfoInAccountManager(BrokerRequest brokerReques
             string homeAccountId = brokerRequest.HomeAccountId;
             string localAccountId = brokerRequest.LocalAccountId;
 
-            if (!string.IsNullOrEmpty(accounts))
+            if (!string.IsNullOrEmpty(accountData))
             {
-                dynamic authResult = JArray.Parse(accounts);
+                dynamic authResult = JArray.Parse(accountData);
```

Comment:
it is used because this is the only object that seems to be able to get the data from the parsed account data. I tried using standard .NET apis for this but they do not always work on Android. Sometimes they throw an error or return null